test/LD.c

```diff
@@ -4,15 +4,21 @@
 #include <time.h>
 #include "../include/justGarble.h"
 
-#define N_allels 10000
-//N_AB
+//two main parameters of LD test
+#define M_allels 10000
```

Comment:
Typo? Should this be `M_alleles`?

Reply:
Based on the recent updates on the documentation (experimental results section), I changed N_allels to M_allels.